src/backend/TrafficCourts/Workflow.Service/Consumers/DisputeApprovedConsumer.cs

```diff
@@ -20,12 +20,11 @@ public DisputeApprovedConsumer(ILogger<DisputeApprovedConsumer> logger, ISubmitD
         }
         public async Task Consume(ConsumeContext<DisputeApproved> context)
         {
+            using var messageIdScope = _logger.BeginScope(new Dictionary<string, object> { { "MessageId", context.MessageId! } });
+
             try
             {
-                if (context.MessageId != null)
-                {
-                    _logger.LogDebug("Consuming message: {MessageId} ", context.MessageId);
-                }
+                _logger.LogDebug("Consuming message of type DisputeApproved");
```

Comment:
The types should be properties, not strings.

Reply:
Done.